src/open_company_web/local_settings.cljs

```diff
@@ -0,0 +1,8 @@
+(ns open-company-web.local-settings)
+
+;; Sentry
+(def local-dsn "https://eab4fe25a2ab4ea1b1d6e10185d63844@app.getsentry.com/48160")
```

Comment:
well we didn't move this out but it will be overwritten when you deploy the repo: https://github.com/belucid/open-company-infrastructure/blob/staging/ansible/web_deploy.yml#L23-L41